paparazzi/src/main/java/app/cash/paparazzi/PaparazziSdk.kt

```diff
@@ -611,7 +620,7 @@ public class PaparazziSdk @JvmOverloads constructor(
         |              android:layout_height="${if (renderingMode.vertAction == RenderingMode.SizeAction.SHRINK) "wrap_content" else "match_parent"}"/>
     """.trimMargin()
 
-    private fun isPresentInClasspath(vararg classNames: String): Boolean {
+    internal fun isPresentInClasspath(vararg classNames: String): Boolean {
```

Comment:
We can do it later but can we move this out of the sdk into a separate place?

Reply:
Yep, I can do this as a follow up